source/app/models/evaluation.rb

```diff
@@ -38,4 +39,8 @@ def set_status_and_date
             self.date_completed = nil
         end
     end
+
+    def send_email_on_completed
```

Comment:
Use `begin-rescue` block to handle potential email delivery errors gracefully. 
This way, if SMTP fails, we log the error instead of crashing.

```
begin
  NotificationMailer.new_evaluation_for_student(evaluatee, self).deliver_later
rescue StandardError => e
  Rails.logger.error("Failed to send evaluation email: #{e.message}")
end
```